app/src/main/res/layout/badge_watched.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Because everything is literally stacked on top of each other it might be better to use a FrameLayout. Makes rendering a bit faster too.